web/src/condense.ts

```diff
@@ -262,14 +281,22 @@ export function condense_and_collapse(elems) {
     }
 }
 
-export function initialize() {
+export function initialize(): void {
     $("#message_feed_container").on("click", ".message_expander", function (e) {
         // Expanding a message can mean either uncollapsing or
         // uncondensing it.
         const $row = $(this).closest(".message_row");
         const id = rows.id($row);
+        if (typeof id !== "number") {
+            return;
+        }
+
         assert(message_lists.current !== undefined);
         const message = message_lists.current.get(id);
+        if (message === undefined) {
```

Comment:
> There is no need to check if the message is undefined or not here is why.
we are checking if we are doing assertation above (line 294) to see if current is undefined or not.
which means, if message_lists exists then there exists a valid id 

here to